WebApi/WebApi/Models/DTO/CategoryDTO.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebApi.Models.DTO
+{
+    public class CategoryDTO
+    {
+        [Key]
+        [Required]
+        public int CategoryID { get; set; }
```

Comment:
jeżeli ten int nie jest nullable to [Required] nie dziala -> musi być nullable, pamietaj o naprawieniu testów w mapperrze które sie wysypia przez to